README.md

```diff
@@ -25,60 +25,119 @@ You can get CIRCL by fetching:
 go get -u github.com/cloudflare/circl
 ```
 
+Alternatively, you can use CIRCL natively by using [Cloudflare Go](https://github.com/cloudflare/go/tree/cf) fork.
```

Comment:
It doesn't contain all packages, does it?

Reply:
initially, we cherry picked packages.
the latest configuration uses an explicit require in go.mod file. so the compiler resolves the set of packages used.

Reply:
Check the `src/vendor/github.com/cloudflare/circl` folder of cfgo. It doesn't contain many packages such as `math/fp448` and such as they aren't used by cfgo itself.